datahub-graphql-core/src/main/java/com/linkedin/datahub/graphql/resolvers/search/SearchUtils.java

```diff
@@ -71,7 +71,6 @@ private SearchUtils() {
           EntityType.CONTAINER,
           EntityType.DOMAIN,
           EntityType.DATA_PRODUCT,
-          EntityType.ROLE,
```

Comment:
Why is this being removed? Did it not actually exist as an entity?

Reply:
we should not remove

Reply:
It was causing the tests to fail because this entity did not exist in the entity registry used for the test. Didn't realize it wasn't just a test list. Let me see how to fix the test without removing it.